Dockerfile

```diff
@@ -1,25 +1,24 @@
-FROM python:3.7.4-alpine3.10
+FROM amazonlinux
 
 RUN mkdir /cdk
 
-COPY ./cdk-packages.txt /cdk/
+COPY ./requirements.txt /cdk/
 COPY ./entrypoint.sh /usr/local/bin/
 
 WORKDIR /cdk
-
-RUN apk -U --no-cache add \
-    bash \
+RUN curl --silent --location https://rpm.nodesource.com/setup_12.x | bash - &&\
```

Comment:
Why are we doing this?

Reply:
this is needed in order to provide a repository for node/npm packages - amazonlinux doesn't have it by default.
The alternative is to use EPEL, but we don't really need to pull down entire EPEL repository just for node.